bot/inviting.py

```diff
@@ -30,6 +30,11 @@ async def on_member_remove(member):
     #Update cache
     invite_cache[member.guild.id] = await member.guild.invites()
 
+#Delete any invite which does not expire after 30 minutes
+async def on_invite_create(invite):
+    if invite.max_age > 1800:
```

Comment:
Is it possible to change the invite age on creation?

Reply:
I don't think it's possible, since invites cannot be modified. What about sending the user a valid, 30-minute invite in DMs after deleting the original invite?

Reply:
Absolutely, that is a solution I prefer

Reply:
I would like to use the `create_invite` function you have made, but the logging creates a problem - it's only meant to log SteamIDs with the use of the `inviter_steamid` parameter. Should the `create_invite` function be modified in a way that it would take either a SteamID or Discord ID and log it correctly?

Reply:
Yea sure, that would work